sdks/python/apache_beam/io/gcp/bigquery_io_metadata.py

```diff
@@ -28,6 +28,11 @@
 _VALID_CLOUD_LABEL_PATTERN = re.compile(r'^[a-z0-9\_\-]{1,63}$')
 
 
+def _sanitize_value(value):
+  """Sanitizes a value into a valid BigQuery label value."""
+  return re.sub('[^\w-]+', '', value.lower().replace('/', '-'))[0:63]
```

Comment:
Ah, just remembered that I checked in helpers in both python and java a python/apache_beam/io/gcp/bigquery_io_metadata.py 

You may want to move the santize function to those files.
_is_valid_cloud_label_value